examples/PasswordExpiryWatcher/README.md

```diff
@@ -0,0 +1,96 @@
+	- Double click on the zip file on macOS/Linux
+5. `PasswordExpiryWatcher_myos_myarch` is the relevant binary
+	- Some may find this is an unwieldy name to type every time. Feel free to rename to your liking, in fact if this is a tool you plan on running frequently we encourage this for ease of use.
+
+> If you'd like to be able to call this binary from an arbitrary directory you can move it to `/usr/local/bin` on Linux and macOS
+>> On macOS the keyboard shortcut `⌘ + Shift + G` with an open Finder window will allow you to directly type in the directory you'd like to access. To move the binary to `/usr/local/bin` with Finder simply use that shortcut and copy/paste `/usr/local/bin` into the dialog box. You can now drag and drop the binary onto the Finder window to easily make it accessible without having to navigate to a specific folder to run it. If you do this we also recommend renaming the binary before moving it to a more memorable name.
+
+##### I have the Go toolchain installed
+> If you don't know what "Go" or "Golang" is we have provided pre-made binaries for your convenience. Installation instructions for these binaries is provided in the previous section
+
+1. Clone the `jcapi` repository (it doesn't matter where)
+	- `git clone https://github.com/TheJumpCloud/jcapi`
+2. Navigate to the `jcapi/examples/PasswordExpiryWatcher` directory
+	- `cd jcapi/examples/PasswordExpiryWatcher`
+3. Run go install
+	- `go install .`
+
+This will install a binary called `passwordExpiryWatcher` to your `$GOBIN`
+
+> If you'd like to be able to call this binary from an arbitrary directory make sure your `$GOBIN` is in your `$PATH` (linux) or `%PATH` (windows)
+
+### To Run
+
+To run this tool you will need to use what is called the "command line". On Windows this is a program called "Power Shell", on macOS it is an app called "Terminal". 
+
+> While running this tool requires no previous experience with either of those programs some might feel wary or nervous working with a tool they don't understand. The following instructions in this section should provide you with all you need to get up and running, but if you would like to learn about the how and why of the command line we highly recommend the excellent (and free!) [Command Line Crash Course by Zed Shaw](http://cli.learncodethehardway.org/). Zed even provides a direct email hotline for users that get stuck. However if you are stuck and would rather just get your users' password expiration dates ASAP please don't hesitate to contact JumpCloud support for assistance running this tool.
+
+This tool only takes two (required) arguments:
+- `key` is your JumpCloud API key
+- `output` is where you'd like to put the resulting CSV file
+
+For example:
+
+macOS/Linux:
+
+`./PasswordExpiryWatcher -key=82105124f2979e28273d4e8dd32b2355c5012837 -output=password_expirations.csv`
+
+Windows:
+
+`./PasswordExpiryWatcher.exe -key=82105124f2979e28273d4e8dd32b2355c5012837 -output=password_expirations.csv`
```

Comment:
I think you can drop the ".exe" suffix for your commands on Windows, so they can all have the exact same instructions if you want.